addons/account/data/mail_template_data.xml

```diff
@@ -131,5 +131,17 @@
             <field name="lang">{{ object.partner_id.lang }}</field>
             <field name="auto_delete" eval="True"/>
         </record>
+
+        <!-- set default invoice template -->
+        <record id="default_invoice_template" model="ir.config_parameter">
```

Comment:
Should be defined in a ``ir_config_parameter_data.xml`` file (do a file per model, it eases understanding the module's structure). Same for other config parameters :)